src/operations/profiles/types.ts

```diff
@@ -62,6 +71,11 @@ export const AccountInfo = t.intersection([
      * The string identifier for the account name.
      */
     name: t.string,
+
+    /**
+     * The account subtype.
+     */
+    subType: AccountInfoSubType,
```

Comment:
I think it's an optional field, judging by the incoming data. Will this satisfy it?

Reply:
yeah. It's fine. At the moment, i'm also setting it is an optional field.